src/test/javascript/spec/component/course/course-exercises.component.spec.ts

```diff
@@ -197,31 +189,31 @@ describe('CourseExercisesComponent', () => {
         ];
 
         // Number of exercises in the course must be 4, since we have added 4 exercises
-        expect(component.course!.exercises!.length).to.equal(4);
+        expect(component.course!.exercises!.length).toEqual(4);
         component.toggleFilters([ExerciseFilter.UNRELEASED]);
 
         // Number of active modeling and text exercises must be 1 respectively, since we have filtered
         // the exercises with release date in the future
-        expect(component.exerciseCountMap.get('modeling')).to.equal(1);
-        expect(component.exerciseCountMap.get('text')).to.equal(1);
+        expect(component.exerciseCountMap.get('modeling')).toEqual(1);
```

Comment:
Try `toBe()` as you compare to a primitive value
```suggestion
        expect(component.exerciseCountMap.get('modeling')).toBe(1);
```